pay-api/src/pay_api/models/invoice.py

```diff
@@ -66,6 +70,7 @@ class Invoice(Audit):  # pylint: disable=too-many-instance-attributes
     receipts = relationship('Receipt', lazy='joined')
     payment_account = relationship('PaymentAccount', lazy='joined')
     references = relationship('InvoiceReference', lazy='joined')
+    corp_type = relationship('CorpType', foreign_keys=[corp_type_code], lazy='select', innerjoin=True)
```

Comment:
Needed to add this relationship, because we were looking at invoice-> payment_line_item -> fee_schedule -> corp_type before?